File: 1.1/DotNetCore-Hello-World-App/tests/UnitTest1.cs
using NUnit.Framework;
using DotNetCore_Hello_World_App;


namespace DotNetCore_Hello_World_App_Test
{
    public class Tests
    {
        Calculate calc = new Calculate();

        [SetUp]
        public void Setup()
        {
            
        }

        [Test]
        public void Test1()
        {
            Assert.Pass();
        }

        [Test]
        public void PassingTest()
        {
            
            Assert.AreEqual(4, calc.Add(2, 2));
        }

        [Test]
        public void FailingTest()
        {
            Assert.AreNotEqual(5, calc.Add(2, 2));
        }
    }
}

File: VB/VB46-Hello-World-App/test/UnitTest1.vb
Imports System.Text
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports VB46_Hello_World_App

<TestClass()> Public Class UnitTest1
    Private _calculate As Calculate = New Calculate

    <TestMethod()> Public Sub PassingTest()

        Assert.AreEqual(4, _calculate.Add(2, 2))
    End Sub

    <TestMethod()> Public Sub FailingTest()

        Assert.AreNotEqual(5, _calculate.Add(2, 2))
    End Sub

    <TestMethod()> Public Sub Test1()

    End Sub

End Class